spec/services/true_layer/importers/import_transactions_service_spec.rb

```diff
@@ -57,8 +57,8 @@
 
         expect(WebMock).to have_requested(:get, expected_request_url)
           .with(query: {
-                  from: now_minus_3_month.iso8601,
-                  to: now.iso8601
+                  from: now_minus_3_month.beginning_of_day.iso8601,
+                  to: now.beginning_of_day.iso8601
                 })
```

Comment:
As we have fixed dates  in the test, can we test here for the actual strings - then that would be a test that the right formatting is taking place

Reply:
I've updated the tests to ensure that the correct format is being produced and sent to true_layer